packages/nodes-base/nodes/Jira/Jira.node.ts

```diff
@@ -678,7 +679,7 @@ export class Jira implements INodeType {
 						}
 						responseData.fields = mappedFields;
 					}
-					returnData.push(responseData);
+					simplifyOutput ? returnData.push(responseData.fields) : returnData.push(responseData);
```

Comment:
@Joffcom not sure we want this. This will mean we lose the `key` and `id` top level fields. The actual `key` isn't referenced directly in the fields structure.